policies/releasestrat.html

```diff
@@ -34,20 +34,6 @@ <h5>
 	  performance improvements and so on. There is no need to
 	  recompile applications to benefit from these features.</p>
 
-	  <p>Binary compatibility also allows other possibilities. For
-	  example, consider an application that wishes to utilize
-	  a new cipher provided in a specific 1.0.x release, but it
-	  is also desirable to maintain the application in a 1.0.0
-	  context.  Customarily this would be resolved at compile time
-	  resulting in two binary packages targeting different OpenSSL
-	  versions. However, depending on the feature, it might be
-	  possible to check for its availability at run-time, thus cutting
-	  down on the maintenance of multiple binary packages. Admittedly
-	  it takes a certain discipline and some extra coding, but we
-	  would like to encourage such practice. This is because we
-	  want to see later releases being adopted faster, because new
-	  features can improve security.</p>
-
```

Comment:
Are we allowed to make changes like this without a vote?

Reply:
I guess not.  Starting a vote now.

Reply:
I don't see this vote anywhere. I think you should remove this change from this PR so that we can get on and push the rest of it.